frameworks/hdfs/universe/marathon.json.mustache

```diff
@@ -14,6 +14,7 @@
     "DCOS_SERVICE_SCHEME": "http"
   },
   "env": {
+    "HDFS_VERSION": "hadoop-2.6.0-cdh5.9.1",
```

Comment:
Kind of a bummer we have to type this out twice, but I guess there's not a way around needing to place the config files at the right location.